studio/local_queue.py

```diff
@@ -45,15 +45,26 @@ def has_next(self):
         is_active, files = self._get_queue_status()
         return is_active and len(files) > 0
 
+    def is_active(self):
+        is_active = os.path.isfile(self.status_marker)
+        return is_active
+
     def clean(self, timeout=0):
-        while self.has_next():
-            self.dequeue()
+        with _local_queue_lock:
```

Comment:
Simply delete all files in the queue folder, do not depend on the presence of queue status marker.